src/net.cpp

```diff
@@ -2837,24 +2837,24 @@ uint64_t CConnman::GetMaxOutboundTarget()
     return nMaxOutboundLimit;
 }
 
-uint64_t CConnman::GetMaxOutboundTimeframe()
+std::chrono::seconds CConnman::GetMaxOutboundTimeframe()
 {
     LOCK(cs_totalBytesSent);
     return nMaxOutboundTimeframe;
 }
 
-uint64_t CConnman::GetMaxOutboundTimeLeftInCycle()
+std::chrono::seconds CConnman::GetMaxOutboundTimeLeftInCycle()
 {
     LOCK(cs_totalBytesSent);
     if (nMaxOutboundLimit == 0)
-        return 0;
+        return std::chrono::seconds::zero();
 
-    if (nMaxOutboundCycleStartTime == 0)
+    if (nMaxOutboundCycleStartTime.count() == 0)
         return nMaxOutboundTimeframe;
 
-    uint64_t cycleEndTime = nMaxOutboundCycleStartTime + nMaxOutboundTimeframe;
-    uint64_t now = GetTime();
-    return (cycleEndTime < now) ? 0 : cycleEndTime - GetTime();
+    std::chrono::seconds cycleEndTime = nMaxOutboundCycleStartTime + nMaxOutboundTimeframe;
+    auto now = GetTime<std::chrono::seconds>();
+    return (cycleEndTime < now) ? std::chrono::seconds::zero() : cycleEndTime - GetTime<std::chrono::seconds>();
```

Comment:
Could reuse `now` here instead of calling `GetTime()` again:
```return (cycleEndTime < now) ? std::chrono::seconds::zero() : cycleEndTime - now;```

Reply:
Thanks. I was going to leave this as a straight refactor, but have  taken your suggestion.